src/pg_to_evalscript/javascript_processes/gte.js

```diff
@@ -0,0 +1,31 @@
+function gte(arguments) {
+  const { x, y } = arguments;
+
+  if (x === undefined) {
+    throw Error("Process gte requires argument x.");
+  }
+
+  if (y === undefined) {
+    throw Error("Process gte requires argument y.");
+  }
+
+  if (x === null || y === null) {
+    return null;
+  }
+
+  if (
+    (typeof x !== "number" && typeof x !== "string") ||
+    (typeof y !== "number" && typeof y !== "string")
+  ) {
+    return false;
```

Comment:
Similar thing here

> If the operands are not [eq](https://docs.openeo.cloud/processes/#eq)ual (see process eq) and any of them is not a number or temporal string (date, time or date-time), the process returns false.

So I guess `gt({"x":"b", "y": "a"})` should return `False` but `gt({"x":"a", "y": "a"})` should return `true`.

Similar for `lte`.

Reply:
was added , but we will have to think of something else if there will be more interdependent processes

Reply:
> but we will have to think of something else if there will be more interdependent processes

What do you mean?